src/core/stdc/stdlib.d

```diff
@@ -81,55 +81,55 @@ else version(CRuntime_Bionic) enum RAND_MAX = 0x7fffffff;
 else static assert( false, "Unsupported platform" );
 
 ///
-double  atof(in char* nptr);
+double  atof(scope const char* nptr);
 ///
-int     atoi(in char* nptr);
+int     atoi(scope const char* nptr);
 ///
-c_long  atol(in char* nptr);
+c_long  atol(scope const char* nptr);
 ///
-long    atoll(in char* nptr);
+long    atoll(scope const char* nptr);
 
 ///
-double  strtod(in char* nptr, char** endptr);
+double  strtod(scope inout(char)* nptr, scope inout(char)** endptr);
 ///
-float   strtof(in char* nptr, char** endptr);
+float   strtof(scope inout(char)* nptr, scope inout(char)** endptr);
 ///
-c_long  strtol(in char* nptr, char** endptr, int base);
+c_long  strtol(scope inout(char)* nptr, scope inout(char)** endptr, int base);
```

Comment:
Sure. dmd doesn't rely on the old signature, it had to workaround it with a cast. See https://github.com/dlang/dmd/pull/6539